LayoutTests/imported/w3c/web-platform-tests/css/css-overflow/scrollbar-gutter-root-both-edges-expected.html

```diff
@@ -34,6 +32,22 @@
   background-color: purple;
   color: white;
 }
+
 </style>
+
+<div id="outer" style="width: 100px; height: 100px; overflow: scroll; left: -200px; position: absolute;">
+    <div id="inner" style="width: 100%; height: 200%;"></div>
+ </div>
+
+ <script>
+        var outer = document.getElementById("outer");
+    var inner = document.getElementById("inner");
+    // Compute scrollbar gutter size and add it as margin to the root element
+    var scrollbarWidthPlusMargin = String((outer.offsetWidth - inner.offsetWidth) + 10) + "px";
+    document.documentElement.style.marginRight = scrollbarWidthPlusMargin;
+    document.documentElement.style.marginLeft = scrollbarWidthPlusMargin;
+
+ </script>
+
 <p id="content">Should not have space around me in the inline axis.</p> 
-<div></div>
\ No newline at end of file
+<div id="abspos"></div>
```

Comment:
You'll want to add an extra blank line to appease the WPT linter

Reply:
Fixed.